backend/graphql/core/src/lib.rs

```diff
@@ -34,6 +36,7 @@ pub trait SelfRequest: Send + Sync {
 pub trait ContextExt {
     fn get_connection_manager(&self) -> &StorageConnectionManager;
     fn get_loader<T: anymap::any::Any + Send + Sync>(&self) -> &T;
+    fn get_loader_v1<T: anymap::any::Any + Send + Sync>(&self) -> &T;
```

Comment:
Just for my understanding... how come you're defining the v1 loader here on the graphql/core ContextExt as well? From what I can see it's only used from the graphql_v1/core ContextExt??